### ISSUE TITLE
Also export the images as tar files in backups folder

### ISSUE THREAD

**Issue by pirate** [funkyfuture/compose-dump#18]
Not all image builds are deterministic, especially if the versions aren't pinned in the compose file.
I propose `compose-dump` should also support saving the built images when doing backups.

Proposed behavior:
```bash
compose-dump ... --images ...
```
Saves all the images used by the project containers into:  
`{host}__{name}__{path_hash}_{date}_{time}/images`

It can using the `docker save --output <image name>.tar <image id>` to do so.

**Comment by funkyfuture** [comment]
yes, this is certainly required for proper archiving. just two remarks:

- each distinct image should be saved in `images/<image_uuid>.tar`
- therefore the `<image_uuid>` must be recorded per service in the `Manifest.yml` of the archive

**Comment by pirate** [comment]
Here's my working solution so far: https://gist.github.com/pirate/265e19a8a768a48cf12834ec87fb0eed
